README.md

```diff
@@ -36,6 +37,7 @@ The following Google style guides live outside of this project:
 <a rel="license" href="https://creativecommons.org/licenses/by/3.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/3.0/88x31.png" /></a>
 
 [cpp]: https://google.github.io/styleguide/cppguide.html
+[csharp]: https://google.github.io/styleguide/csharp-style.html
```

Comment:
This link is .html and the other file is .md

Reply:
The R guide does this as well and seems to work - must be some GitHub magic.

Can amend or not as you like.